chall04/phakakos.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import sys
+import fileinput
+
+def err(num):
+        if (num == 0):
+                print("Give size as a number")
+                exit(1)
+        if (num == 1):
+                print("Invalid map input. Please check")
+                exit(2)
+        if (num == 2):
+                print("Map width does not match")
+                exit(3)
```

Comment:
err(1) ; err(0) ; err(2) are only used once each. So I would avoid creating a function like this one and suggest to use .

```python
sys.exit("your message")
```

https://docs.python.org/2/library/sys.html#sys.exit